src/store.ts

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import persistantStateSlice from './modules/oh-indicators-list/store/slice';
+
+export const store = configureStore({
+  reducer: {
+    indicators: persistantStateSlice,
```

Comment:
Renamed to `indicators` cause `persistent` is too generic. 
Open to a discussion